### ISSUE TITLE
Stoneblock 3 is extremly laggy

### ISSUE THREAD

**Issue by ASH12Lost** [FTBTeam/FTB-Modpack-Issues#5605]
### Modpack

FTB StoneBlock 3

### Modpack version

1.10

### Has the pack been modified

No

### Describe the bug

If i load into the game all is good and nice but after a few seconds of peace the lags begin, it never ends. I mean i have a ryzen 7 3800x, 16Gb RAM, RTX3060 all should work, but even at minimum settings the lags dont stop

### Steps to reproduce

1. Open Skyblock3
2. Multiplayer -> join server
3. all peace
4. a few seconds past and the lags begin

### Expected behaviour

that the game runs smooth

### Additional information

i hope you can help me thx

**Comment by Gaz492** [comment]
If you log into single player and create a new world is it still laggy for you?

May I ask what type of lag you are having? Is it FPS lag (you can check your fps by pressing the F3 key) or is it lag where it takes a while for blocks to break and opening chests/doors is delayed

**Comment by ASH12Lost** [comment]
Yes when I play singleplayer its still laggy, and its fps lags. It feels like its loading something in the backround but I cant say what and no its not the light overlay. Thx for your help mate

**Comment by Gaz492** [comment]
Ok, did this just start happening out of nowhere or, has this been an issue from the very beginning of playing SB3?

If its been an issue from the beginning, does the issue happen on other modpacks?

**Comment by ASH12Lost** [comment]
I have played the modpack for a while and there was no issue, but after a month or two ive started a new world with a friend and since that day I have these problems. No its only on Stoneblock 3
I have also uninstalled it and reinstalled it but nothing changes.

**Comment by ASH12Lost** [comment]
Can it be my Graphic Card Drivers? I have the newest one but before it wasnt that better. But idk im still searching for the problem

**Comment by Thijs226** [comment]
i have the same problem

**Comment by muhammetozmen** [comment]
RTX4050
DDR5-4800 32 GB RAM
Ryzen 5
Still super laggy, i'm not having that much lag in any other modpacks.

**Comment by Sza1367** [comment]
if you have many drawers in your world it will start to lag your entre world

**Comment by VileZX** [comment]
Just to give a better understanding of the situation, something in this modpack is refreshing every 1 second which causes the game to freeze for half a second. It's not enough to stop people from playing, but it is enough to be annoyingly noticeable.

**Comment by DinnerBeef** [comment]
Can you upload a spark report?
You can do this with this command `/spark profiler start --timeout 60`
